docs/application/dotnet/guides/nui/window.md

```diff
@@ -313,6 +313,32 @@ child.Unparent();
 > If parent's window stack is changed by calling `Raise()` or `Lower()`, the child window is located on top of the parent again.
 >
 
+### Window type
+
+NUI window supports several window types. For the list of supported types, see [WindowType] (application/dotnet/api/TizenFX/latest/api/Tizen.NUI.WindowType.html).
```

Comment:
Remove the space between the square bracket and normal brackets, otherwise it does not recognize as hyperlink
```suggestion
NUI window supports several window types. For the list of supported types, see [WindowType](application/dotnet/api/TizenFX/latest/api/Tizen.NUI.WindowType.html).
```